packages/core-js/utils/hardhat.js

```diff
@@ -7,7 +7,7 @@ async function getContractAST({ sourceName, contractName }) {
     : output.sources[sourceName].ast;
 }
 
-async function getAllContractASTs(hre) {
+async function getAllContractASTs() {
```

Comment:
Removed the `hre` passed by params top be consistent with other internal tools (which use global `hre` definition)

Reply:
Disagree here.

Internal utils in the deployer can expect a globally injected hre, but generic tools in core-js cannot. This is why I specified it as a parameter.

Reply:
Agree with @ajsantander

Reply:
fixed!